periphboard/board.go

```diff
@@ -39,6 +40,10 @@ func newBoard(
 	conf resource.Config,
 	logger golog.Logger,
 ) (board.Board, error) {
+	if _, err := host.Init(); err != nil {
```

Comment:
I'm inclined to put this up in `init()` instead of here, so it only runs once at the beginning, rather than every time we create a new board. What do you think?

Reply:
I will leave here for now - it could make sense if someone changed their board settings at the OS level and then wanted to re-configure?  Not sure but I don't think it will cause problems here.